framework/console/controllers/FixtureController.php

```diff
@@ -415,13 +415,36 @@ private function findFixtures(array $fixtures = [])
         $foundFixtures = [];
 
         foreach ($files as $fixture) {
-            $foundFixtures[] = basename($fixture, 'Fixture.php');
+            $relativeName = $this->getFixtureRelativeName($fixture);
+            $foundFixtures[] = $relativeName;
         }
 
         return $foundFixtures;
     }
 
     /**
+     * Calculates $fixture's name relatively to $templatePath.
+     * Basically, strips getFixturePath() and 'Fixture.php' prefix from fixture's full path
+     * @see getFixturePath()
+     * @param $fixture string Full fixture path
+     * @return string Relative fixture name
+     *
+    */
+    private function getFixtureRelativeName($fixture) {
+        // $fixturesPath is normalized to unix format in getFixturesPath()
```

Comment:
Negative.
`$fixture` is a string; one of the strings that `FileHelper::findFiles()` returns [here](https://github.com/yiisoft/yii2/pull/13553/files/719f6aee4856d2e5bbd9b44100955d9e34f94573#diff-96190cbb10ccc75fed657c73f25ae920L414).